### PR TITLE
Add proxy support to host.v1/host.v2. Fixes #1397

### LINKED ISSUE #1397: HTTP Proxy suport for host.v1/host.v2 config types
Initial version doesn't need to support authentication

### PR DESCRIPTION
**Pull request by plorenz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,11 +2,32 @@
...
+
+## Proxy Support in host.v1/host.v2
+
+`host.v1` and `host.v2` configurations may now specify a proxy to use. 
+Currently only HTTP Connect proxies which don't require authentication are supported.
+
+**Example using `host.v1`**
+
+{
+    "address": "localhost",
```
Comment (scareything) [inline CHANGELOG.md:15]: would the example make more sense if `address` (and perhaps also `proxy.address`) weren't "localhost", but remote/lan IPs?

**Review by scareything — COMMENTED** [review]
are you going to update the rendered json files?

File: CHANGELOG.md (hunk 1 above)
Comment (plorenz) [reply CHANGELOG.md]: Done

**Comment by plorenz** [comment]
> are you going to update the rendered json files?

Done

**Review by ekoby — APPROVED** [review]

**Pull request merged**